### PR TITLE
Feat detail

### PR DESCRIPTION
**Pull request by gabriela99**
Detail page has been added and connected to the dashboard list of resources. Mostly UI changes have been implemented here. On the dashboard you can 'see details'. There you will see the detail page, where you can click on 'go to resource', and this will take you to the URL provided. The create page was edited a bit here, and changes from feat_create have been pulled in. 

screenshot 2018-11-28 at 14 20 09

screenshot 2018-11-28 at 14 20 25

screenshot 2018-11-28 at 14 20 45

### CODE REVIEW

File: learningresource/templates/learningresource/detail.html [hunk 1]
```diff
@@ -18,50 +18,102 @@ <h1 style="margin: auto; width: 600px;"> {{ resource.title }} </h1>
...
-                {{ resource.pub_date }}
-            </div>
-            
-            <div class="form-group">
-                <label style="font-weight: bold">URL</label>
-                {{ resource.url }} 
-            </div>
+        <dl class="row">
+            <dt class="col-sm-3">Created by</dt>
+            <dd class="col-sm-9">{{ resource.user }}</dd>
```
Comment (dani-santos-code) [inline learningresource/templates/learningresource/detail.html:25]: Nice!

File: learningresource/templates/learningresource/detail.html [hunk 2]
```diff
@@ -12,13 +12,44 @@
...
+
+            <div class="checkbox">
+                <label style="font-weight: bold"> Is it free?
+                    {{ instance.is_free }}
+                </label>
+            </div>
+        </div>
     {% endif %}
 {% endfor %}
 </div>
```
Comment (dani-santos-code) [inline learningresource/templates/learningresource/detail.html]: Well done!

File: dashboard/templates/dashboard.html [hunk 3]
```diff
@@ -52,7 +52,7 @@ <h4>{{ learningResource.short_title }}</h4>
                 <h4>{{ learningResource.title }}</h4>
               {% endif %}
               <div style="padding: 2px;">{{ learningResource.department }}</div>
-              <a href="{{ resource.detail }}" 
+              <a onclick="window.location='{% url 'detail' learningResource.id %}';"
```
Comment (dani-santos-code) [inline dashboard/templates/dashboard.html]: Why do you need the window.location here? I just tried `<a onclick="{% url 'detail' learningResource.id %}"`.  It's more Django-like and works the same way! \o/

File: dashboard/templates/dashboard.html [hunk 4]
```diff
@@ -6,7 +6,7 @@
 <main role="main">
 <section class="jumbotron text-center bg-gray">
     <div class="container">
-        <h1 class="jumbotron-heading" style="line-height:80%;"><img src="{% static 'logo2.png' %}" height=40/> Resource Center</h1>
+        <h1 class="jumbotron-heading" style="line-height:80%; font-weight: 200" ><img src="{% static 'logo2.png' %}" height=39 style="padding-bottom: 7px"/> RESOURCE CENTER</h1>
```
Comment (dani-santos-code) [inline dashboard/templates/dashboard.html:9]: Good job here. Looks better! :)

**Review by dani-santos-code — APPROVED** [review]

File: dashboard/templates/dashboard.html (hunk 3 above)
Comment (faymarie) [reply dashboard/templates/dashboard.html]: I think it's a browser issue. To be on the save side it should be `onclick="javascript:window.location.href='{% url 'detail' learningResource.id %}';"`. Check this out: https://stackoverflow.com/questions/10252809/onclick-window-location-href-wont-work-in-chrome-or-safari

**Review by faymarie — APPROVED** [review]
Good job. Will copy some things for the user/profile settings pages. :)

File: dashboard/templates/dashboard.html (hunk 3 above)
Comment (gabriela99) [reply dashboard/templates/dashboard.html]: Fixed it with commit #784226b

**Pull request merged**